test/e2e-tests.sh

```diff
@@ -40,6 +40,21 @@ header "Setting up environment"
 install_pipeline_crd
 
 failed=0
+function alpha_gate() {
```

Comment:
it looks like the function `set_feature_gate` has an existing section for ` if [ "$gate" == "alpha" ];` - does it maek sense to do this setup there instead? or make a new function for alpha setup in general and move the logic from `set_feature_gate` into there as well?